code/modules/power/floodlight.dm

```diff
@@ -0,0 +1,106 @@
+
+/obj/structure/floodlight_frame
+	name = "floodlight frame"
+	desc = "A bare metal frame looking vaguely like a floodlight. Requires wrenching down."
+	max_integrity = 100
+	obj_integrity = 100
+	icon = 'icons/obj/lighting.dmi'
+	icon_state = "floodlight_c1"
+	density = TRUE
+	var/state = FLOODLIGHT_NEEDS_WRENCHING
+
+/obj/structure/floodlight_frame/attackby(obj/item/O, mob/user, params)
+	if(istype(O, /obj/item/weapon/wrench) && (state == FLOODLIGHT_NEEDS_WRENCHING))
+		to_chat(user, "<span class='notice'>You secure the [src].</span>")
+		anchored = TRUE
```

Comment:
This is inconsistent, frame gets wrenched, but the actual floodlight starts unanchored.